### PR TITLE
Ipa foedag integration

### PR DESCRIPTION
**Pull request by w0lek**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -3,7 +3,8 @@
 	url = https://github.com/os-fpga/yosys_verific_rs.git
 [submodule "FOEDAG_rs"]
 	path = FOEDAG_rs
-	url = https://github.com/os-fpga/FOEDAG_rs.git
+	#url = https://github.com/os-fpga/FOEDAG_rs.git
+	url = https://github.com/w0lek/FOEDAG_rs.git
```
Comment (volodymyrkochyn) [inline .gitmodules:7]: This shouldn't be changed

File: .gitmodules [hunk 2]
```diff
@@ -3,7 +3,8 @@
 	url = https://github.com/os-fpga/yosys_verific_rs.git
 [submodule "FOEDAG_rs"]
 	path = FOEDAG_rs
-	url = https://github.com/os-fpga/FOEDAG_rs.git
```
Comment (volodymyrkochyn) [inline .gitmodules:6]: This shouldn't be changed

**Comment by volodymyrkochyn** [comment]
Pending https://github.com/os-fpga/FOEDAG_rs/pull/677

**Comment by volodymyrkochyn** [comment]
Before merge need to change default IPA value to disable

**Comment by alaindargelas** [comment]
@ravikiranchollangi @chungshien-chai please create another PR in lieu of this one to merge FOEDAG_rs

**Pull request closed without merging**